docs/reference/search.md

```diff
@@ -0,0 +1,102 @@
+# Searching in InvenioRDM
+_Introduced in InvenioRDM v13_
+
+### InvenioRDM Suggest API
+
+The suggest API endpoint (`/api/{resource}?suggest={search_input}`) provides an interface for real-time search suggestions. It leverages OpenSearch's `multi_match` query to search across multiple fields within a specified index, returning relevant suggestions based on user input.
+
+#### Endpoint Structure
+
+**URL:** `/api/{resource}?suggest={search_input}`
+**Method:** GET
+
+Each index in InvenioRDM can have its own configuration to customize how the suggest API behaves. This includes defining which fields are searchable and other settings provided by the `multi_match` query API.
+
+## How to use suggest API?
+
+InvenioRDM's Suggest API is designed to provide search suggestions by using a `multi_match` query. It can be configured for all the indices using the `SuggestQueryParser` class that can be imported from `invenio-records-resources` module. The fields are analyzed using custom analyzers at index time which apply filters like `asciifolding` for accent search and `edge_ngram` to generate tokens of all lengths according to the filter settings to power `prefix matching` search capabilities in the dropdown UI fields.
+
+Check the [official documentation](https://opensearch.org/docs/2.0/opensearch/ux/) and [Reference](#reference) for more context on the `edge_ngram` filter and custom analyzers.
+
+### When to Use the Suggest API
+
+- **Typo Tolerance & Auto-completion:** Helps correct typos (using `fuzziness` at search time analyzing) and completes partial inputs.
+- **Large, Diverse Datasets:** Useful for datasets with a wide variety of terms, like names or titles.
+- **Pre-query Optimization:** Reduces unnecessary searches by suggesting relevant terms.
+
+### When Not to Use the Suggest API
+
+- **Small or Specific Datasets:** Less beneficial for well-defined datasets.
+- **Performance Constraints:** Because the suggest API creates large amounts of token using the `edge_ngram` filter, it is important to observe how it affects the index size.
```

Comment:
```suggestion
- **Performance Constraints:** Because the suggest API creates large amounts of tokens using the `edge_ngram` filter, it is important to observe how it affects the index size.
```